images/python2/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM python:2.7
```

Comment:
If we're going to the "python2-debian" route, the Dockerfile here might as well renamed to "Dockerfile-debian" to avoid confusion.

Reply:
Hmm, if that’s the route we want to go, why not tuck this Dockerfile in a folder named “Debian” instead? IMO, it looks neater.

Reply:
Well, we don't want too many folders. Besides, adding `-<whatever>` is already being used in other projects for alternate versions/tags